announcementsAPI.json

```diff
@@ -363,12 +370,54 @@
         "properties": {
           "name": {
             "type": "string",
-            "nullable": true
+            "nullable": false
+          },
+          "sex":{
+            "$ref": "#/components/schemas/Sex"
           },
           "species": {
+            "type": "string",
+            "nullable": false
+          },
+          "breed": {
+            "type": "string",
+            "nullable": true
+          },
+          "birthday": {
+            "type": "string",
+            "format": "date-time"
+          },
+          "description": {
             "type": "string",
             "nullable": true
           },
+          "photo": {
+            "type": "string",
+            "format": "byte",
+            "nullable": true
+          }
+        },
+        "additionalProperties": false
+      },
+      "PetResponse": {
```

Comment:
If we are already suggesting PetResponse I would suggest to add some way to access a photo - maybe a link? To be discussed.